src/components/CryptoCoinItem/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React, {useEffect, useState} from 'react';
+import {ActivityIndicator, ScrollView} from 'react-native';
+import Avatar from '../Avatar';
+import {
+  Container,
+  ContainerLeft,
+  ContainerRight,
+  ContainerSymbol,
+  TextPercentage,
+  TitleBold,
+  TitleNormal,
+  Wrapper,
+  OuterContainer,
+  TextPercentageValue,
+} from './styles';
+import {CryptoCoin} from '../types';
+import {fetchCryptoData} from './cryptoAPI';
+import Icon from 'react-native-vector-icons/Feather';
+
+export const CryptoCoinItem = () => {
+  const [isLoading, setLoading] = useState<boolean>(true);
```

Comment:
there is no need to declare <boolean> it will get the type of the value, in basic cases like strings, numbers, boolean, etc